.eslintrc

```diff
@@ -0,0 +1,40 @@
+{
+  "extends": [
+    "airbnb",
+    "prettier",
+    "prettier/react"
+  ],
+  "plugins": ["react", "prettier"],
+  "parser": "babel-eslint",
+  "env": {
+    "node": true,
+    "es6": true,
+    "mocha": true,
+    "browser": true,
+    "commonjs": true,
+    "jest": true
+  },
+  "rules": {
+    "new-cap": 0,
+    "import/prefer-default-export": "off",
+    "prettier/prettier": "error",
+    "import/no-unresolved": 0,
+    "no-plusplus": 0,
+    "react/sort-comp": 0,
+    "react/prop-types": 0,
+    "no-unused-expressions": 0,
+    "import/no-named-as-default": 0,
+    "consistent-return": 0,
+    "prefer-destructuring": 0,
+    "no-shadow": 0,
```

Comment:
This rule can help us write clear and unconfusing code, and can help prevent the confusion between global and local variables. Any reason as to why you are turning it off?